s3repo/model.py

```diff
@@ -0,0 +1,200 @@
+"""Model for working with the repositories on S3."""
```

Comment:
Just to understand layout:

```
. (repository: tarantool-repositories)
├── app.py
├── config.default
├── s3repo
│   ├── __init__.py
│   ├── model.py
│   ├── package.py
│   └── view.py
```

So, the application is name as 'tarantool-repositories'. It has a module, 's3repo'. How that everything can evolve in theory? I mean, what modules may be added and so on.

Whether we'll support source tarballs uploading / listings within this service?

Should not it be named in a more general way, like 'download.tarantool.org'?

Reply:
> Just to understand layout:
> 
> ```
> . (repository: tarantool-repositories)
> ├── app.py
> ├── config.default
> ├── s3repo
> │   ├── __init__.py
> │   ├── model.py
> │   ├── package.py
> │   └── view.py
> ```
> 
> So, the application is name as 'tarantool-repositories'. It has a module, 's3repo'. How that everything can evolve in theory? I mean, what modules may be added and so on.
> 

Can be added the modules to work with local or some other type of storage (for example).

> Whether we'll support source tarballs uploading / listings within this service?

I don't see why not if we want to add this functionality.

> 
> Should not it be named in a more general way, like 'download.tarantool.org'?

If you talk about the repo name - yes, it will be renamed when moving to the "tarantool" organization.
If you talk about the service name - I renamed it to "RWS".

Reply:
Okay.